python/image.bzl

```diff
@@ -88,4 +88,4 @@ def py_image(name, base=None, deps=[], layers=[], **kwargs):
     index += 1
 
   app_layer(name=name, base=base, entrypoint=['/usr/bin/python'],
-            binary=binary_name, layers=layers)
+            binary=binary_name, layers=layers, visibility=visibility)
```

Comment:
This is missing a definition of `visibility`.

Reply:
@mattmoor fixed.